contracts/staking/tests/ERC20TokenStakingManagerTests.t.sol

```diff
@@ -45,6 +45,25 @@ contract ERC20TokenStakingManagerTest is PoSValidatorManagerTest {
         posValidatorManager = app;
     }
 
+    function testCompleteEndValidation() public override {
+        bytes32 validationID = _setUpInitializeEndValidation({
+            nodeID: DEFAULT_NODE_ID,
+            subnetID: DEFAULT_SUBNET_ID,
+            weight: DEFAULT_WEIGHT,
+            registrationExpiry: DEFAULT_EXPIRY,
+            blsPublicKey: DEFAULT_BLS_PUBLIC_KEY,
+            registrationTimestamp: DEFAULT_REGISTRATION_TIMESTAMP,
+            completionTimestamp: DEFAULT_COMPLETION_TIMESTAMP
+        });
+
+        uint256 balanceBefore = token.balanceOf(address(this));
+
```

Comment:
we should also add checks for the `expectCall`s

Reply:
addressed in https://github.com/ava-labs/teleporter/pull/527/commits/6decd640de94d330ac9435f346f4614743fb9fe1

Reply:
addressed for Native in https://github.com/ava-labs/teleporter/pull/527/commits/fa3931d25b785a8d63223b59c3989062791ecc02